index.html

```diff
@@ -80,41 +84,110 @@ <h2 class="display-4">Who we are</h2>
   <section>
     <div class="container">
       <div class="row align-items-center">
-        <div class="col-md-4">
+        <div class="col-md-4 order-2">
           <div class="p-5 d-none d-md-block">
-            <img class="img-fluid rounded-circle" src="./images/history.jpg" alt="">
+            <img class="img-fluid rounded-circle " src="./images/NY.jpg" alt="">
           </div>
         </div>
-        <div class="col-md-8">
+        <div class="col-md-8 order-1">
           <div class="p-5">
-            <h2 class="display-4">Proud of our history</h2>
-            <p>This <a href="https://medium.com/@x_inaka/from-company-to-community-fa3c7ab041bf">amazing blogpost</a>
-              explains our origins and history after 7+ years of existence. Please take a look.</p>
+            <h2 class="display-4">Contact Us</h2>
+            <p>If you have questions, suggestions or comments about any of these projects (or if you want to join us)
+              please contact the dev team
+              at <a href="mailto:inakos@googlegroups.com">inakos@googlegroups.com</a></p>
           </div>
         </div>
       </div>
     </div>
+    </div>
   </section>
 
+  
+
   <section>
+    <div class="container" id="events">
+      <div class="col-md-12">
+        <div class="p-5">
+          <h2 class="display-4">Events</h2>
+          <p>This are our events.</p>
```

Comment:
```suggestion
          <p>Besides maintaining open-source projects, we also organize or contribute to the organization of several meetups and other recurring events. Check them out in the list below!</p>
```

Reply:
I think this change may be lost… since I made it on index.html instead of the source of it, right?
Not 100% familiar with node.js to know, sorry.

Reply:
Now I suggested the change on the template below.